docs/reference/adobe/experience/customerJourneyManagement/messageprofile.schema.json

```diff
@@ -47,6 +47,11 @@
                         "apns": "Apple Push Notification service",
                         "fcm": "Firebase cloud messaging service"
                     }
+                },
+                "https://ns.adobe.com/experience/customerJourneyManagement/emailChannelContext/namespace": {
```

Comment:
Minor comment, It will be more readable if we add this new field after line 41, so that all related fields are together in this file.